libraries/Schedule/src/circular_queue/circular_queue_mp.h

```diff
@@ -0,0 +1,184 @@
+/*
```

Comment:
Same comments as for sp class.

Reply:
Same opinion on my side as above. Maybe later?